model/annotations/task_annotations_test.go

```diff
@@ -159,3 +159,42 @@ func TestMoveSuspectedIssueToIssue(t *testing.T) {
 	assert.Equal(t, "different user", annotationFromDB.Issues[0].Source.Author)
 	assert.Equal(t, "someone new", annotationFromDB.Issues[1].Source.Author)
 }
+
+func TestPatchIssue(t *testing.T) {
```

Comment:
do we maybe want to add a test that has a note or metadata?

Reply:
done